### PR TITLE
Reorganize by domains

### PR DESCRIPTION
**Pull request by sowbiba**

### CODE REVIEW

File: tests/News/NewsBuilderTest.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+use Mockery\Adapter\Phpunit\MockeryTestCase;
+use PrestaShop\Module\Mbo\News\News;
+use PrestaShop\Module\Mbo\News\NewsBuilder;
+
+class NewsBuilderTest extends MockeryTestCase
+{
+    public function testBuildWillReturnAValidNewsInstance()
+    {
+        $date = 'Mon, 28 Feb 2022 11:24:05 +0100';
```
Comment (intraordinaire) [inline tests/News/NewsBuilderTest.php:31]: Better to use a DateTime or Carbon object no ?

File: tests/News/NewsBuilderTest.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            ->allowMockingUnknownTypes()
+            ->setMethods([
+                'displayDate',
+                'truncateString',
+            ])
+            ->getMock();
+
+        $tools
+            ->method('displayDate')
+            ->willReturn('2022-02-28 11:24:05');
```
Comment (intraordinaire) [inline tests/News/NewsBuilderTest.php:48]: So we can use it here

File: tests/News/NewsBuilderTest.php [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            $link,
+            $countryIsoCode,
+            $contextMode
+        );
+
+        $this->assertInstanceOf(News::class, $news);
+
+        $this->assertSame([
+            'date' => '2022-02-28 11:24:05',
+            'title' => 'CMS PrestaShop : guide d&#039;installation manuelle de votre boutique ecommerce',
```
Comment (intraordinaire) [inline tests/News/NewsBuilderTest.php:74]: Why did not use the variables created earlier ?

Comment (sowbiba) [reply tests/News/NewsBuilderTest.php:74]: to show how elements are transformed. You'll see that the strings are different

**Review by intraordinaire — APPROVED** [review]

File: tests/News/NewsBuilderTest.php (hunk 3 above)
Comment (intraordinaire) [reply tests/News/NewsBuilderTest.php:74]: Ahah missed that one indeed

**Pull request merged**